fuzzers/kirenenko/discard.list

```diff
@@ -0,0 +1,93 @@
+fun:jsimd_can_rgb_ycc=discard
```

Comment:
Another, simpler, option would be avoiding asm code all together, so we wouldn't need this discard list for dfsan. For libjpeg-turbo, we can do that with `-DWITH_SIMD=0`. https://github.com/libjpeg-turbo/libjpeg-turbo#memory-debugger-pitfalls

One question whether we should use that flag (1) with all fuzzers ie. in the libjpeg build script or (2) let fuzzer.py config's decide to add the flag or not.

I'd recommend (2), because it'd allow using MSAN with all fuzzers (MSAN also doens't work well with asm). Wdyt @jonathanmetzman ?